### DISCUSSION TITLE
Error: Command failed: npm install --production --only=prod

### DISCUSSION THREAD

**Discussion by neetesh1996** [serverless/serverless#10694] [Q&A]
`apple@Apples-MacBook-Pro` be-v1 % serverless deploy --stage dev          
Running "serverless" from node_modules

Deploying `stack` to stage dev (ap-south-1)

[ LayersPlugin ]: => common
... ○ Downloading ./package.json from bucket...
... ○ ./package.json AWS profile "profiler" doesn't seem to be configured.
... ○  Changes identified ! Re-installing...
... ∅ [warning] ".npmrc" file does not exists!
... ∅ [warning] "yarn.lock" file does not exists!
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
.git can't be found (see https://git.io/Jc3F9)
npm ERR! code 1
npm ERR! path /Users/apple/Desktop/be-v1/.serverless/layers/nodejs
npm ERR! command failed
npm ERR! command sh -c husky install

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/apple/.npm/_logs/2022-02-13T08_55_36_762Z-debug.log

✖ Stack satckename failed to deploy (3s)
Environment: darwin, node 14.18.1, framework 3.2.0 (local) 3.1.1v (global), plugin 6.0.0, SDK 4.3.1
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Error: Command failed: npm install --production --only=prod
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
.git can't be found (see https://git.io/Jc3F9)
npm ERR! code 1
npm ERR! path /Users/apple/Desktop/be-v1/.serverless/layers/nodejs
npm ERR! command failed
npm ERR! command sh -c husky install

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/apple/.npm/_logs/2022-02-13T08_55_36_762Z-debug.log

    at checkExecSyncError (child_process.js:790:11)
    at execSync (child_process.js:863:15)

**Comment by pgrzesik** [comment]
Hello @neetesh1996, in the output you've pasted there's an error that `.git` can't be found along with a link - it seems like your `husky` config might be invalid.
